<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration_form_doctor</class>
 <widget class="QMainWindow" name="Registration_form_doctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 240);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Full name*</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_doctorFirstName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
     <property name="text">
      <string>First</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_doctorLastName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
     <property name="text">
      <string>Last</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Email*</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_doctorEmail">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>age*</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_doctorAge">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
     <property name="minimum">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Years of experience*</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_doctorExp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>319</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Speciality*</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_speciality">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Or another:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_newSpec">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Working days:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_Monday">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Monday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_Tuesday">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_Wednesday">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>480</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_Thursday">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>510</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Thursday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_Friday">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>480</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Friday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_Saturday">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>510</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Saturday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_Sunday">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>490</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Sunday</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Start of working day:</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_start">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>550</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>End of working day:</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_end">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>570</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>640</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
